modules/nf-core/polypolish/polish/meta.yml

```diff
@@ -0,0 +1,76 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "polypolish_polish"
+description: Polishing genome assemblies with short reads.
+keywords:
+  - assembly polishing
+  - genome polishing
+  - ont
+tools:
+  - "polypolish":
+      description: "Polishing genome assemblies with short reads."
+      homepage: "https://github.com/rrwick/Polypolish"
+      documentation: "https://github.com/rrwick/Polypolish/wiki"
+      tool_dev_url: "https://github.com/rrwick/Polypolish"
+      doi: "10.1099/mgen.0.001254"
+      licence: ["GPL3"]
+      identifier: biotools:polypolish
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - fasta:
+        type: file
+        description: FASTA file
```

Comment:
Please put more info what this FASTA file should contain, e.g. is it a refrence genome?